.circleci/main.yml

```diff
@@ -1163,7 +1163,60 @@ jobs:
                 echo "regular release."
                 gh release create --draft ${CIRCLE_TAG} --title ${CIRCLE_TAG:1} ./dist-pach/pachctl/pachctl_${CIRCLE_TAG:1}* ./dist-pach/mount-server/mount-server_${CIRCLE_TAG:1}* --generate-notes
             fi
+  console-release-draft:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - gh/setup:
+          version: 2.13.0
+      - run:
+          name: create github console release draft
+          command: |
+            if [[ $CIRCLE_TAG == *"-"* ]];
```

Comment:
Is there any reason you put this command inline instead of in an sh script? I don't have a preference, just asking for my own knowledge.

Reply:
good question. I really don't know what the preference has been, endless shell scripts or endless lines of yaml 😆 ideally would love to break this config out and leverage shell scripts to further break it down.